pkgs/development/python-modules/marshmallow/default.nix

```diff
@@ -21,7 +19,5 @@ buildPythonPackage rec {
     sha256 = "35ee2fb188f0bd9fc1cf9ac35e45fd394bd1c153cee430745a465ea435514bd5";
```

Comment:
Would you mind adding yourself as a maintainer?

Reply:
Done